requirements/edx-sandbox/base.in

```diff
@@ -18,11 +18,11 @@
 -r shared.txt                       # Dependencies in common with LMS and Studio
 chem                                # A helper library for chemistry calculations
 matplotlib==2.2.4                   # 2D plotting library
-numpy==1.7.2                        # Numeric array processing utilities; used by scipy
+numpy==1.16.5                       # Numeric array processing utilities; used by scipy
 pyparsing==2.2.0                    # Python Parsing module
 random2                             # Implementation of random module that works identically under Python 2 and 3
-scipy==0.14.0                       # Math, science, and engineering library
-sympy==0.7.1                        # Symbolic math library
+scipy==1.2.1                        # Math, science, and engineering library
+sympy==1.4                          # Symbolic math library
 git+https://github.com/edx/openedx-calc.git@e9b698c85ad1152002bc0868f475f153dce88952#egg=calc==0.4
```

Comment:
Possibly for a separate PR, but should opened-calc come from PyPI after this PR: https://github.com/edx/edx-platform/pull/23378/files

Reply:
Yea, I'll do that in a follow-up.